src/pages/Profile/Profile.css

```diff
@@ -1,8 +1,41 @@
 .profile-page {
-    height: 100vh;
-    width: 100vw;
-    display: flex;
-    align-items: center;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
```

Comment:
If you zoom in and out on your page, you can see that the profile page isn't centered. Since the parent component, profile-page, already has `display: flex` applied to it, you can easily center it vertically using `justify-content: center`.